db_schema/migrations/12_hoster_support.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO migrations (migration_id, migration_name) VALUES (12, 'genericHoster');
+
+CREATE TABLE IF NOT EXISTS hosterPodcastMetadata (
+  account_id INTEGER NOT NULL,
+  hoster_id MEDIUMINT UNSIGNED NOT NULL,
+  date DATE NOT NULL,
+  name VARCHAR(2048) NOT NULL,
+  -- Date is not part of the primary key
+  -- because we only want to store the latest data
+  PRIMARY KEY (account_id, hoster_id)
+);
+
+CREATE TABLE IF NOT EXISTS hosterPodcastMetrics (
+  account_id INTEGER NOT NULL,
+  hoster_id MEDIUMINT UNSIGNED NOT NULL,
+  start DATETIME NOT NULL,
+  end DATETIME NOT NULL,
+  dimension ENUM(
+    'downloads',
+    'platforms',
+    'clients',
+    'sources'
+  ) NOT NULL,
+  subdimension VARCHAR(255) DEFAULT '' NOT NULL,
```

Comment:
we have many rows in this table, so I would try to get rid of string fields here.